### DISCUSSION TITLE
Amazing, how does playground manage to load remote ts definition files

### DISCUSSION THREAD

**Discussion by zjfjiayou** [esm-dev/esm.sh#733] [General]
playground is very Amazing, even can load the remote ts definition, may I ask if there is any plan to develop this part of the code? In addition, is there such a plug-in in vscode (currently I am using deno's plug-in, but my project is not deno's project)?

**Comment by ije** [comment]
since esm.sh transforms dts for deno runtime, os i hecked the `ts-worker` of [monaco-editor](https://github.com/microsoft/monaco-editor) to fetch the types from esm.sh.
i'm planning to open soruce it but did not get time yet. maybe create a vscode extension as well.

**Reply by zjfjiayou** [reply]
f it can be open source really great, I really want to learn the implementation logic here, very much looking forward to early open source, thank you

**Reply by al3xnag** [reply]
I'd be interested to look at the `ts-worker` for monaco too.
